addon/system/target.js

```diff
@@ -268,19 +267,27 @@ var Target = Ember.Object.extend(Ember.Evented, {
 
       var eventManager = this.eventManager;
       eventManager.mouseup = bind(this, 'mouseUp');
-      $(document).on('mouseup touchend', eventManager.mouseup);
+      $element.on('mouseup touchend', eventManager.mouseup);
```

Comment:
I believe this event was required on document to handle closing event when user clicks out of pop-over.